src/core/verbs/global-verbs.api.ts

```diff
@@ -0,0 +1,34 @@
+import { defaultFullVerbs, defaultSelectedVerbs } from './global-verbs.storage';
+import Lockr from 'lockr';
+
+export interface VerbEntityApi {
+  infinitive: string;
+  past: string;
+  participle: string;
+  translation: string;
+}
+
+let fullVerbCollection: VerbEntityApi[] = defaultFullVerbs;
+let selectedVerbCollection: string[] = Lockr.get(
+  'selectedVerbs',
+  defaultSelectedVerbs
+);
+
+// Simulating AJAX Call
+export const loadFullVerbCollection = async (): Promise<VerbEntityApi[]> =>
+  Promise.resolve(fullVerbCollection);
+
+export const loadSelectedVerbCollection = async (): Promise<string[]> => {
+  selectedVerbCollection = Lockr.get('selectedVerbs', defaultSelectedVerbs);
```

Comment:
'selectedVerbs'

This should be enclosed in a const on top of this file (We are using the same magic string in line 22 and 33, easy to introduce a typo and a hard to pinpoint bug)